### DISCUSSION TITLE
PiJuice Does Not Install on 2022-01-28-raspios-bullseye-arm64

### DISCUSSION THREAD

**Discussion by poseidonphotos** [PiSupply/PiJuice#819] [Q&A]
Hi,

I am running the latest 64 bit version of Raspberry Pi OS (2022-01-28-raspios-bullseye-arm64) on Raspberry Pi 4 (8GB) and on top of that I am trying to install PiJuice. 

When I try to install with the command "sudo apt-get install pijuice-gui", I get an error: "E: Unable to locate package pijuice-gui"

I have referenced the instructions mentioned in below discussion as well but to no avail. Trying to manually install the 1.7 packages ("git clone https://github.com/PiSupply/PiJuice/blob/master/Software/Install/pijuice-gui_1.7_all.deb") throws an error ("fatal: repository 'https://github.com/PiSupply/PiJuice/blob/master/Software/Install/pijuice-gui_1.7_all.deb/' not found")
https://github.com/PiSupply/PiJuice/discussions/805#discussioncomment-1894099

Could someone please shed some light? I am fairly new to the whole realm of raspberry pi.

**Comment by tvoverbeek** [comment]
Do not use git clone. Go to the relevant URL in your browser and just select the Download button.
Proper packages usable on all 3 OS versions (buster 32-bit, bullseye 32-bit and bullseye 64-bit) will be uploaded to github this week.
Will take a bit longer before they appear in the appropriate repositories.

**Reply by poseidonphotos** [reply]
Thanks for the response. 

I tried to do as per your instruction, but when attempting to install I get an error message:
"Failed to install file. A package could not be found that allows the action to complete. More information is available in the detailed report."

What am I doing wrong here?

**Reply by tvoverbeek** [reply]
You first need to install pijuice-base before pijuice-gui.
pijuice-base in turn depends on python3-urwid.
First install python3-urwid from the bullseye 64-bit repository
THen manually install pijuice-base, followed by manually installing pijuice-base.
Right now you still need to do the multi-arch thing since the included executables are 32-bit only.

**Reply by poseidonphotos** [reply]
Ah ha! Voila! Works now. Thank you so much!

**Reply by tvoverbeek** [reply]
Since today (Feb 11) the 1.8 packages are in the relevant repositories. So you can install the normal way.

**Reply by poseidonphotos** [reply]
Thanks!
